cli/azd/pkg/pipeline/pipeline_manager.go

```diff
@@ -379,7 +379,7 @@ func (pm *PipelineManager) Configure(ctx context.Context) (result *PipelineConfi
 
 	// config pipeline handles setting or creating the provider pipeline to be used
 	ciPipeline, err := pm.ciProvider.configurePipeline(
-		ctx, gitRepoInfo, infra.Options, additionalSecrets, additionalVariables)
+		ctx, gitRepoInfo, infra.Options, additionalSecrets, additionalVariables, pm.configOptions)
```

Comment:
It took me awhile to realize that the result of `SecretsAndVars` is actually "the final list of secrets and variables to set". Does it make sense to name the variables here to something else?